src/it/scala/org/ergoplatform/it/PrunedDigestNodeSyncSpec.scala

```diff
@@ -0,0 +1,65 @@
+package org.ergoplatform.it
+
+import java.io.File
+
+import akka.japi.Option.Some
+import com.typesafe.config.Config
+import org.asynchttpclient.util.HttpConstants
+import org.ergoplatform.it.container.{IntegrationSuite, Node}
+import org.scalatest.FreeSpec
+
+import scala.async.Async
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class PrunedDigestNodeSyncSpec extends FreeSpec with IntegrationSuite {
+
+  val approxTargetHeight = 10
+  val blocksToKeep: Int = approxTargetHeight / 2
+
+  val localVolume = s"$localDataDir/digest-node-sync-spec/data"
+  val remoteVolume = "/app"
+
+  val dir = new File(localVolume)
+  dir.mkdirs()
+
+  val minerConfig: Config = nodeSeedConfigs.head
+    .withFallback(miningDelayConfig(10000))
+    .withFallback(specialDataDirConfig(remoteVolume))
+  val digestConfig: Config = digestStatePeerConfig
+    .withFallback(blockIntervalConfig(8000))
+    .withFallback(prunedHistoryConfig(blocksToKeep))
+    .withFallback(nonGeneratingPeerConfig)
+    .withFallback(nodeSeedConfigs(1))
+
+  // Testing scenario:
```

Comment:
This test scenario actually does not check, that node did not applied full blocks with `height < {targetHeight - blocksToKeep}`. So it is still possible, that node have downloaded all the blocks and removed blocks at `0-{targetHeight - blocksToKeep}` after that.
Test should check that Digest node initialized at block `{targetHeight - blocksToKeep}` without applying first blocks,

Reply:
@catena2w, checks added